providers/implementations/storemgmt/winstore_store.c

```diff
@@ -23,7 +23,9 @@
 #include "prov/implementations.h"
 #include "prov/bio.h"
 #include "file_store_local.h"
-
+#ifdef __CYGWIN__
+#include <windows.h>
```

Comment:
Please add one space between `#` and include

Reply:
done in https://github.com/openssl/openssl/pull/21006/commits/aa85bd19afa3445d7dbb947564ebecb2148e8fc4